index.html.slim

```diff
@@ -29,9 +29,9 @@ description: The simple, modern & productive way to build apps and infrastructur
           img.mobile(src="http://static.jboss.org/images/hero/hero_mobile_480x500_devnation_06112015.jpg" alt="DevNation: Watch the keynotes, live and on demand")
           img.desktop(src="http://static.jboss.org/images/hero/hero_desktop_1175x400_devnation_06112015.jpg" alt="DevNation: Watch the keynotes, live and on demand")
       .large-24.slide
-        a( href="https://www.redhat.com/apps/store/middleware/jboss_enterprise_application_platform.html")
-          img.mobile(src=cdn("http://static.jboss.org/images/hero/hero_mobile_480x500_eapwebstore_051215.jpg") alt="")
-          img.desktop(src=cdn("http://static.jboss.org/images/hero/hero_desktop_1175x400_eapwebstore_051215.jpg") alt="")
+        a( href="http://vertx.io/")
+          img.mobile(src=cdn("http://static.jboss.org/images/hero/hero_mobile_480x500_vertx3_06252015.jpg") alt="")
```

Comment:
You need to put alt tag values in. This messes up SEO if we miss them off. Use the text from the image.